<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Input/Output parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>281</width>
           <height>71</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>91</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Decay constants of progenies</string>
         </property>
         <property name="title">
          <string>Decay constants</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>-1</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>l1 [1/s]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="text">
                 <string>0.003729</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>l2 [1/s]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="text">
                 <string>0.000431</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>l3 [1/s]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="text">
                 <string>0.0005805</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>71</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>371</width>
           <height>91</height>
          </size>
         </property>
         <property name="title">
          <string>Concentrations [Bq/l]</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>c1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_4">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>c2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_5">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>c3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_6">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>241</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QCommandLinkButton" name="commandLinkButton_5">
         <property name="text">
          <string>Progenies time dependence</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>81</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_8"/>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="statusTip">
             <string>Amount of filtered gas</string>
            </property>
            <property name="text">
             <string>Gas volume [l]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="statusTip">
             <string>Time of radon filtration through the filter</string>
            </property>
            <property name="text">
             <string>Filtration time [s]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="statusTip">
             <string>Efficiency to grap progenies</string>
            </property>
            <property name="text">
             <string>Filter efficiency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_9"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCommandLinkButton" name="commandLinkButton_2">
      <property name="text">
       <string>Compute concentrations</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCommandLinkButton" name="commandLinkButton_4">
      <property name="text">
       <string>Show input data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_database"/>
    <addaction name="actionSave_database"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionAblout_Qt">
   <property name="text">
    <string>Ablout Qt</string>
   </property>
  </action>
  <action name="actionOpen_database">
   <property name="text">
    <string>Open database</string>
   </property>
  </action>
  <action name="actionSave_database">
   <property name="text">
    <string>Save database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
